src/main/java/com/untamedears/humbug/Humbug.java

```diff
@@ -1531,6 +1531,73 @@ public void onPlayerBucketEmptyEvent(PlayerBucketEmptyEvent e) {
       }
     }
   }
+  
+  @BahHumbug(opt="drop_ores_as_ore", def="true")
+  @EventHandler(priority = EventPriority.HIGHEST, ignoreCancelled = true)
+  public void orebreak(BlockBreakEvent e) {
+	  if(!config_.get("drop_ores_as_ore").getBool()) {
+		  return;
+	  }
+	  if (isOre(e.getBlock().getType())) {
+		  e.setCancelled(true);
```

Comment:
From reading a bit on the bukkit forums it's not possible. You can only get the drops of a block but can't change them. Maybe firing another block break event with the same player and block will do the job. Considering you can't change the drops with the current API I don't think it will drop anything.